processors/HNL.py

```diff
@@ -203,6 +194,65 @@
        leptonCollection = lambda event: event.tightMuon[0] 
      )
 )
+
+analyzerChain.append(
+    JetMetUncertainties(
+	era = globalOptions["year"],
+	globalTag=jecTags[globalOptions["year"]],
+	jerTag = jerTags[globalOptions["year"]],
+	isData= args.isData 
+    )
+)
+
+
+if isMC:
+	
+	for systName,collection in [
+        	("nominal",lambda event: event.jets_nominal),
+	        ("jerUp",lambda event: event.jets_jerUp),
+	        ("jerDown",lambda event: event.jets_jerDown),
+	        ("jesTotalUp",lambda event: event.jets_jesUp["Total"]),
+	        ("jesTotalDown",lambda event: event.jets_jesDown["Total"]),
+    	]:
+
+        	analyzerChain.append(
+            		JetSelection(
+                		inputCollection=collection,
+				leptonCollection=lambda event: event.tightMuon,		
+	                	outputName="selectedJets_"+systName,
+		                storeKinematics=['pt','eta'],
+        			globalOptions=globalOptions
+			)
+	        )
+
+	
```

Comment:
should also add the lepJet selection here for each variation

Reply:
ok  true